go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/fsouza/go-dockerclient v1.4.1
 	github.com/go-sql-driver/mysql v1.6.0
 	github.com/gogo/protobuf v1.3.2
+	github.com/golang/protobuf v1.5.2
```

Comment:
where is this getting used? We probably want to be using gogo proto referenced above

Reply:
Looks like it’s in `pachgen`:

```
etc/proto/pachgen/main.go
9:      descriptor "github.com/golang/protobuf/protoc-gen-go/descriptor"
10:     plugin "github.com/golang/protobuf/protoc-gen-go/plugin"

etc/proto/pachgen/transaction_client.go
11:     descriptor "github.com/golang/protobuf/protoc-gen-go/descriptor"
12:     plugin "github.com/golang/protobuf/
```